src/js/stores/AppsStore.js

```diff
@@ -295,7 +314,7 @@ QueueStore.on(QueueEvents.CHANGE, function () {
 AppDispatcher.register(function (action) {
   switch (action.actionType) {
     case AppsEvents.REQUEST_APPS:
-      storeData.apps = processAppsAndEmptyGroups(action.data.body);
+      storeData.apps = processAppsAndGroups(action.data.body);
```

Comment:
Please also rename the property as well as the store getter, should be something like `items` as it includes groups and apps.